app/main/posts/modify/post-data-editor.directive.js

```diff
@@ -242,10 +242,6 @@ function PostDataEditorController(
             attributes.map(function (attr) {
                 // Create associated media entity
                 if (attr.input === 'upload') {
-                    var media = {};
```

Comment:
why is this being removed? (just curious, I think I am a little lost 😬 ) is this now handled with the viewChange /

Reply:
This was not used, made the conclusion its forgotten remains from a previous refactoring.